integration-test/src/main/scala/com/cloudera/livy/test/framework/Cluster.scala

```diff
@@ -48,20 +49,26 @@ trait Cluster {
 
   def doAsClusterUser[T](task: => T): T
 
-  lazy val hadoopConf = {
-    val conf = new Configuration(false)
-    configDir().listFiles().foreach { f =>
-      if (f.getName().endsWith(".xml")) {
-        conf.addResource(new Path(f.toURI()))
-      }
-    }
-    conf
-  }
+  lazy val coreSiteConf = loadConf(s"${configDir().getCanonicalPath}/core-site.xml")
+
+  lazy val yarnSiteConf = loadConf(s"${configDir().getCanonicalPath}/yarn-site.xml")
```

Comment:
Is there an issue with creating the yarn client when all configuration is loaded? The previous code does not hardcode config file names, which I prefer.